src/blocks/AboutUsHeroBlock/index.tsx

```diff
@@ -92,7 +92,7 @@ const onPlay = ({
     video.pause()
     video.currentTime = 0
     video.play()
-    video.webkitEnterFullscreen()
+    ;(video as any).webkitEnterFullscreen()
```

Comment:
i know it looks super-weird but otherwise TS/JS will interpret this as `video.webkitEnterFullscreen()().webkitEnterFullScreen()` which is something competely different